src/pages/gen1/[platform]/build-a-backend/auth/add-social-provider/index.mdx

```diff
@@ -739,6 +739,74 @@ function App() {
 </Block>
 </BlockSwitcher>
 
+### Redirect URLs
+
+You can provide multiple _Sign in Redirect URI(s)_ & _Sign out redirect URI(s)_.
+
+If you have multiple redirect URI inputs, you'll need to pass them in your Amplify configuration. For example:
+
+```javascript
+Amplify.configure({
+  Auth: {
+    Cognito: {
+      loginWith: {
+        oauth: {
+          redirectSignIn: [
+            'myDevApp://'
+          ],
+          redirectSignOut: [
+            'myDevApp://',
+            'myProdApp://'
+          ],
+          ...oauthConfig
+        }
+      },
+      ...userPoolConfig
+    }
+  }
+});
+```
+
+#### Option to specify a redirect URL upon signOut
```

Comment:
```suggestion
#### Specifying a redirect URI on sign out
```

Also, do we have consistent language on URI vs URL for this feature? E.g. the previous section. Also capitalized vs lower case consistency.

Reply:
Good catch, I think we can keep it as URI upper case as that's how it's on the Cognito [docs](https://docs.aws.amazon.com/cognito/latest/developerguide/authorization-endpoint.html#:~:text=be%20an%20absolute-,URI,-.).